src/Components/HeaderPanel/HeaderPanel.tsx

```diff
@@ -1,22 +1,22 @@
-import React, { ReactNode } from "react";
+import React from "react";
 import Nav from "components/Nav/Nav";
 
 import bemCssModule from "bem-css-modules";
 import { default as HeaderPanelStyles } from "./HeaderPanel.module.scss";
 import Header from "components/Header/Header";
 
 interface Props {
-  actionButton: ReactNode;
+  titleButton: string;
 }
 
 const style = bemCssModule(HeaderPanelStyles);
 
-const HeaderPanel: React.FC<Props> = ({ actionButton }) => {
+const HeaderPanel: React.FC<Props> = ({ titleButton }) => {
   return (
     <div className={style()}>
       <Header />
       <Nav />
-      {actionButton}
+      <button className={style("button")}>{titleButton}</button>
```

Comment:
It's ok but previous idea was better. Keep in mind that the code should be easy to extend. Also keep in mind previous implementation is much clearer because you pass only one prop (whole button), now you probably need to pass more props (e.g. title, callback,?icon, ?disabled)